UnityWeld_Editor/AnimatorParameterBindingEditor.cs

```diff
@@ -0,0 +1,337 @@
+                        }
+                    case AnimatorControllerParameterType.Trigger:
+                        {
+                            selectedPropertyType = typeof(AnimatorParameterTrigger);
+                            break;
+                        }
+                    default:
+                        {
+                            selectedPropertyType = null;
+                            break;
+                        }
+                }
+            }
+            else
+            {
+                if (selectedIndex < 0)
+                {
+                    selectedPropertyType = null;
+                    return;
+                }
+
+                switch (properties[selectedIndex].type)
+                {
+                    case AnimatorControllerParameterType.Bool:
+                        {
+                            selectedPropertyType = typeof(bool);
+                            break;
+                        }
+                    case AnimatorControllerParameterType.Float:
+                        {
+                            selectedPropertyType = typeof(float);
+                            break;
+                        }
+                    case AnimatorControllerParameterType.Int:
+                        {
+                            selectedPropertyType = typeof(int);
+                            break;
+                        }
+                    case AnimatorControllerParameterType.Trigger:
+                        {
+                            selectedPropertyType = typeof(AnimatorParameterTrigger);
+                            break;
+                        }
+                    default:
+                        {
+                            selectedPropertyType = null;
+                            break;
+                        }
+                }
+            }
+        }
+
+        /// <summary>
+        /// Check whether each of the properties on the object have been changed from the value in the prefab.
+        /// </summary>
+        private void UpdatePrefabModifiedProperties()
+        {
+            var property = serializedObject.GetIterator();
+            // Need to call Next(true) to get the first child. Once we have it, Next(false)
+            // will iterate through the properties.
+            property.Next(true);
+            do
+            {
+                switch (property.name)
+                {
+                    case "viewAdapterTypeName":
+                        viewAdapterPrefabModified = property.prefabOverride;
+                        break;
+
+                    case "viewAdapterOptions":
+                        viewAdapterOptionsPrefabModified = property.prefabOverride;
+                        break;
+
+                    case "viewModelPropertyName":
+                        viewModelPropertyPrefabModified = property.prefabOverride;
+                        break;
+
+                    case "uiPropertyName":
```

Comment:
There is no "uiPropertyName" field on AnimatorParameterBinding so this is unnecessary. You should add cases for `animatorParameterName` and `animatorParameterType` though.

Actually seeing this makes me realise that since I renamed a bunch of fields on the other binding classes, I'll need to update this function in the other custom editors too. Thanks!